src/v3/src/components/Link/Link.tsx

```diff
@@ -36,23 +40,31 @@ const Link: UISchemaElementComponent<{
 
     onSubmitHandler({
       params: actionParams,
+      includeData: Boolean(includeData),
       actionFn,
       step,
+      stepToRender,
     });
   };
 
+  const hasAction = actionFn !== undefined || step !== undefined;
+
   return (
     <Box marginTop={4}>
       {
-        actionFn !== undefined ? (
-          <LinkMui
-            // eslint-disable-next-line no-script-url
-            href="javascript:void(0)"
-            onClick={onClick}
-          >
-            {getLabelName(label)}
-          </LinkMui>
-        ) : <LinkMui href={href}>{getLabelName(label)}</LinkMui>
+        <LinkMui
+          data-se={uischema.scope}
+          // eslint-disable-next-line no-script-url
+          href={hasAction ? 'javascript:void(0)' : href}
+          // eslint-disable-next-line react/jsx-props-no-spreading
+          {...(hasAction && { onClick } )}
+          // eslint-disable-next-line react/jsx-props-no-spreading
+          {...(dataType && { 'data-type': dataType } )}
```

Comment:
Is this necessary for the `Link` type? I know it was used on `Button` - but maybe still needed for tests.

Reply:
Yeah i was trying to keep it in parity to be safe for testing, but I think I can trim some unnecessary ones.